data/hai/hai reveal 2 start.txt

```diff
@@ -0,0 +1,486 @@
+			`	You notice several men nearby looking intently at Ooonem. The two of you run towards your ship and make it to cover before the men start shooting. Ooonem pulls a grenade out of her cloak and throws it at the men. Gas explodes outward and the men drop to the ground clutching their throats, dead in seconds.`
+			`	"I had better leave these behind," she says, dropping a few gas grenades on the ground. "I took them off of the pirates a few days ago. They're quite illegal." You notice a label on the grenades: "Sarin Gas: twenty-meter radius." This was a nerve gas grenade that uses one of the older varieties of the gas. They're cheap and plentiful in pirate space.`
+			`	You run into your ship and immediately take off.`
+				launch
+			label decline
+			`	The woman runs off into the shadows. You doubt you'll ever see her again.`
+				decline
+	on complete
+		log `Rescued a kidnapped Hai researcher, Ooonem, from Stormhold and returned her to Makerplace. She was tricked into leaving Hai space for a fake business deal, and the merchant who agreed to take her there kidnapped her instead. Men have been hunting her since she escaped. She doesn't know the real identities of the kidnappers.`
+		payment 225000
+		"hai slave prereq" ++
+		dialog `Ooonem is relieved to return to <planet>. She leads you to her family's home where you're treated to a bountiful dinner. The cuisine is entirely human-Hai fusion dishes. The dessert is accompanied by most delicious spice of all: <payment>.`
+	npc
+		government "Bounty Hunter that Won't Enter Hai Space"
+		personality plunders disables unconstrained
+		fleet "Large Core Pirates" 1
+		fleet "Small Core Pirates" 1
+
+
+mission "Hai Rescue: Meroot and Lucy Hint"
+	landing
+	name "Deep Crevasses"
+	description "There could be kidnappers or kidnapped Hai hiding on <destination>."
+	source
+		near "Zaurak" 4
+		government "Republic" "Syndicate"
+		attributes "spaceport"
+		not attributes "station"
+		not planet "Canyon"
+	destination Canyon
+	to offer
+		"combat rating" > 50
+		has "Unwanted Cargo: done"
+		has "Hai Leaks Response 6: offered"
+		has "event: slightly lost"
+		not "Hai Rescue: Meroot and Lucy: offered"
+		not "event: hai-human treaty signed"
+		random < 80
+	on offer
+		conversation
+			`As you come in to land at the spaceport, you listen to the usual chatter of captains coming and going. You don't hear anything specific to Hai, but when the punchline of a joke is "deep crevasses" on <planet>, it occurs to you that there would be the perfect place for both illicit activities and escapees to hide out under the radar for a long time. It's probably worth a look...`
+				accept
+
+mission "Hai Rescue: Meroot and Lucy"
+	landing
+	name "Return Meroot and Lucy"
+	description "Transport the scientists you rescued to <destination>. Payment is <payment>."
+	source Canyon
+	destination Newhome
+	passengers 2
+	repeat 3
+	to offer
```

Comment:
This mission is missing a condition to have it only offer if you're in the storyline. If that's intentional, then this needs rewritten as it assumes you're in the storyline, as there's no explanation as to why there's a Hai here and/or why the player suspects that what they spot is a Hai.

Reply:
Two of these missions can offer as long as you've done the mission where you return the Hai stowaway on your ship. This makes it possible for you to have encountered a small number of Hai who have got stuck and need to get home (once you already know the Hai exist and have helped one of them back home) as foreshadowing for the events of this story.